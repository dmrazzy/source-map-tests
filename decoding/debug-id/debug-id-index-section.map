{
  "version": 3,
  "file": "app.js",
  "sections": [{
    "offset": {
      "line": 0,
      "column": 0,
    },
    "map": {
      "version": 3,
      "file": "component1.js",
      "sources": [],
      "mappings": "",
      "debugId": "a938a92f-3074-41f7-bfdb-1038430a983c"
    }
  }, {
    "offset": {
      "line": 1,
      "column": 0,
    },
    "map": {
      "version": 3,
      "file": "component2.js",
      "sources": [],
      "mappings": "",
      "debugId": "8cf3d888-fc9e-457d-a999-6ffef384b736"
    }
  }]
}