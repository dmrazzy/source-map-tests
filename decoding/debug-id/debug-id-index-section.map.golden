{
  "file": "app.js",
  "sources": [],
  "mappings": [],
}